src/makeRobot.js

```diff
@@ -39,6 +40,62 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name,
+    version,
+    wheels,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return `name: ${this.name}, `
+           + `chip version: ${this.version}, `
+           + `wheels: ${this.wheels}`;
+    },
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    get robot() {
+      return { ...robot };
+    },
+    goForward(distance = 1) {
```

Comment:
I would name this parameter as `step` to be more meaningful
```suggestion
    goForward(step = 1) {
```